wiki/Tournaments/OMMT/3/en.md

```diff
@@ -0,0 +1,481 @@
+- Disqualification from the entire tournament.
+
+### Qualifiers procedure
+
+1. The referee will notify the player 10-15 minutes before the lobby is scheduled to start. Once all the players are present or another 10 minutes has passed since the scheduled time, the lobby will start.
+2. Players will play the mappool in order from Stage 1-5 twice. After the first cycle, players will be given a 5 minute break before continuing with the 2nd cycle which follows the same procedure. Players are allowed to skip this break if all players present in the lobby agrees to start the 2nd run right away.
+
+### Match procedure
+
+1. The referee will notify the player 10 minutes before the match is scheduled to start. Once the player is ready they must notify the referee and they will invite the player to the lobby.
+2. Protecting & Banning:
+   - The player will be required to roll for **protecting and banning maps** for a total of **protecting 2 maps and banning 1 map**. The player with the highest roll will be allowed to protect a map of their choice first, as well being able to ban first, this is a bit complicated so please don't hesitate to ask in the Discord server if you do not understand this explaination.
+   - A player **cannot** protect 2 maps of the same type (ex: player 1 protects hb1, now player 1 cannot protect any of the remaining hybrids), this rule applies to each person individually.
+3. During the protect/ban/pick phases each player will be given a 2 minute time to choose their maps, if players end up taking too long, the ref will roll a number corresponding to the number of maps in the mappool (minus tb, example the mappool has 11 maps so the ref will !roll 10), to randomly choose a map for them. This is to ensure that a match doesn't taking too long as it does not benefit anyone.
+4. Each player will receive one "timeout" of two minutes per match, to be used as extra time to pick a beatmap or to rest. The timeout may only be used by the player whose is picking the next map, the only exception being during tiebreaker scenarios where both players are given a two minute timeout regardless.
+5. Players cannot pick or use a provocative warmup to ensure that the livestream is family friendly to watch. Players are not required to have a warm up map to be played, but if they do they must send their warmups to the referee at least 5 minutes before the match begins, but the earlier the better.
+6. Duration of warm up maps must be under 4 minutes, but there is no star limit.
+
+### Additional procedure
+
+1. Players can reschedule to a later qualifiers lobby if they missed their scheduled lobby.
+2. In case of a situation where the player doesn't show on match time, the match will be delayed by 10 minutes. If the player still does not show up the winner of the match will automatically be the other player present. To avoid this please check schedules properly and do an emergency reschedule beforehand if needed but with a valid reason.
```

Comment:
```suggestion
2. In case of a situation where a player doesn't show up on time, the match will be delayed by 10 minutes. If the player still does not show up, the other player present will automatically be declared the winner of the match. To avoid this, please check schedules properly and, if necessary, do an emergency reschedule beforehand with a valid reason.
```